dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -54,9 +54,13 @@ const decidePresentationFormat = ({
 		linkFormat.design === ArticleDesign.LiveBlog ||
 		linkFormat.design === ArticleDesign.Gallery ||
 		linkFormat.design === ArticleDesign.Audio ||
+		linkFormat.theme === ArticleSpecial.SpecialReport ||
 		linkFormat.design === ArticleDesign.Video
 	)
-		return containerFormat;
+		return {
+			...containerFormat,
+			theme: ArticlePillar.News,
+		};
```

Comment:
Do we need to override the theme, here? If the container was different, we probably would want to respect that.